operator/pkg/utils/utilities.go

```diff
@@ -34,3 +34,37 @@ func SearchMap(mapToSearch map[string]interface{}, key string) map[string]interf
 
 	return make(map[string]interface{})
 }
+
+func FindValueIndexFromStringArray(a []string, v string) int {
```

Comment:
`IndexOf`?

Reply:
Yeah, we might be able to replace this. This function is not new, it was moved here from reconcile_racks.go.

Reply:
There appears to be no generic IndexOf function, unless I've missed something.

Reply:
I was suggesting we rename it

Reply:
Would you be content with `IndexOfString`? I don't want a generic name for a non-generic function in a generic package.

Reply:
👍